ui/lib/sync/addon/routes/secrets.ts

```diff
@@ -22,23 +21,20 @@ export default class SyncSecretsRoute extends Route {
   @service declare readonly store: StoreService;
 
   async fetchActivatedFeatures() {
+    // The read request to the activation-flags endpoint is unauthenticated and root namespace
+    // but the POST is not which is why it's not in the NAMESPACE_ROOT_URLS list
     return await this.store
       .adapterFor('application')
-      .ajax('/v1/sys/activation-flags', 'GET')
+      .ajax('/v1/sys/activation-flags', 'GET', { unauthenticated: true, namespace: null })
       .then((resp: ActivationFlagsResponse) => {
```

Comment:
Wondering if I should match what Noelle is doing in the Client Counts and return empty array if this does error for some reason. That way we will be consistent in how we're handling this

Reply:
Yeah, that's a good idea. we run `includes` on the model arg activatedFeatures which is why I think somewhere I was returning an empty array.

Reply:
yeah, now that we don't need to track the adapter error i think we're probably just safe to catch the error and add 
 `if (DEBUG) console.error(error)`. that'll help notify us of errors when they do occur but not break the UI.

my thinking was that if the API call fails for some reason, we might as well assume they have no activated features. but i dunno, maybe it's better to assume the opposite? i don't have a good gauge on what conditions would cause that API call to fail. 😕